packages/Path-View-Core.package/SScrollPane.class/instance/containsDiagram.st

```diff
@@ -0,0 +1,5 @@
+accessing
+containsDiagram
+	
+	 containsDiagram ifNil: [^ false].
```

Comment:
Here we have to make the decision whether we use lazy initialization or by nico suggested one.